azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/PackageMojo.java

```diff
@@ -108,6 +117,44 @@ protected URL getTargetClassUrl() throws Exception {
         return outputDirectory.toURI().toURL();
     }
 
+    /**
+     * This method will search the output directory which configured in the Maven Dependency Plugin.
+     * Maven Dependency Plugin will extract the runtime jars into the output directory,
+     * and these jars will be used for annotation search using reflection.
+     * @return URLs for the runtime needed jars
+     */
+    protected List<URL> getDependencyArtifactUrls() {
+        final List<URL> urlList = new ArrayList<>();
+        final Plugin dependencyPlugin = this.getProject().getPlugin(DEPENDENCY_PLUGIN_KEY);
+        if (dependencyPlugin != null) {
```

Comment:
so it requires the pom.xml to have "copy-dependencies" goal? is it a walk around?

Reply:
Emm, that is a work around. But I just find a better solution for that.

Code is updated, sorry for the confusion.